doc/content/getting-started/aws/ecs/database-operations/_index.md

```diff
@@ -91,6 +93,8 @@ tti-lw-stack,is-db,create-tenant,--id=your-tenant-id,--name=Your Tenant Name
 {{</ tabs/tab >}}
 {{</ tabs/container >}}
 
+> **NOTE:** Replace `--id=your-tenant-id` and `--name=Your Tenant Name` with your own.
```

Comment:
This need not be explained really.

Reply:
Yeah, but it is in all other steps, so why not in this one?

Reply:
It's redundant. That's what the `<>` notation indicates. Better @htdvisser looks at this since he's the author.

Reply:
I don't really care, but I definitely think we can make it more consistent. In this guide alone we already mix `--id=your-tenant-id`, `--secret=CLIENT_SECRET` and `--tenant-id,<your-tenant-id>`. I can imagine other guides also have different styles. Maybe @benolayinka can assist with making all of this more consistent?

Reply:
i added a section to CONTRIBUTING.md about this - if possible, define the variables in a separate command, i.e
```
$ TENANT_ID="your-tenant-id"
$ CLIENT_SECRET="your-client-secret"
$ TENANT_NAME="your-tenant-name"
$ tti-lw-stack is-db create-tenant --id=$TENANT_ID --name=$TENANT_NAME
```

https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/104/files

Reply:
@michalborkowski96 so the action here is to replace `your-tenant-id` etc. with `$TENANT_ID` etc. I'm fine with keeping the "replace" note, because I can imagine all the quotes we have will screw with the shell expanding the variables.